src/input-backend/main.rs

```diff
@@ -58,7 +59,11 @@ fn main() -> Result<(), zbus::Error> {
 		.udev_assign_seat("seat0")
 		.expect("Could not assign seat0");
 
-	let pollfd = PollFd::new(input.as_raw_fd(), PollFlags::POLLIN);
+
+    let fd = input.as_raw_fd();
+    assert!(fd != -1);
+    let borrowed_fd = unsafe{ BorrowedFd::borrow_raw(input.as_raw_fd())};
+    let pollfd = PollFd::new(&borrowed_fd, PollFlags::POLLIN);
```

Comment:
Did the previous impl break anything?

Reply:
It would not compile otherwise. Note that my rust is not the strongest, this compiled and worked for me.

Reply:
@ErikReider now ported to nix 0.29 and with cargo fmt